src/main/java/it/pagopa/pm/gateway/client/util/Constants.java

```diff
@@ -15,6 +15,6 @@
 
     public static final String TOKEN =System.getProperty("bancomatPay.client.token");
 
```

Comment:
we can use Value spring annotation to get properties? https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/annotation/Value.html

Reply:
Will be changed in next commit